CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#3524](https://github.com/open-telemetry/opentelemetry-python/pull/3524))
 - Handle `taskName` `logrecord` attribute
   ([#3557](https://github.com/open-telemetry/opentelemetry-python/pull/3557))
+- Add Proxy classes for logging
```

Comment:
Please move to Unreleased section

Reply:
Will do once I know your preference on the comment below re exposing the classes in __init__

Reply:
This is currently under `Version 1.22.0/0.43b0 (2023-12-15)`, can you move it to Unreleased section?

Reply:
Done, thanks